app/javascript/components/works/Filters.jsx

```diff
@@ -69,15 +70,16 @@ class Filters extends PureComponent {
                 <div className="checkbox-container">
                   {Object.keys(filters[type]).map(item => (
                     <div className="mb2 checkbox-item" key={item}>
-                      <label for={`checkbox-${item}`}>
+                      <label className = "ttc dib flex" for={`checkbox-${item}`}>
```

Comment:
<img width="412" alt="Screen Shot 2019-03-18 at 8 43 33 PM" src="https://user-images.githubusercontent.com/13279205/54578908-94c40900-49be-11e9-80db-d8db08941ecf.png">

Hmm this is still a little wonky for me for some reason :( 

@ethanlee16 is it showing up fine for you?

Reply:
@thumn I think the solution is to give the text a width manually:
```
<span className="filter-item">{convertSnakeCase(item)}</span>
```

```
.filter-item {
    width: 146px;
}
```